pom.xml

```diff
@@ -385,6 +399,11 @@
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Please don't introduce the scala dependency, it's hard to maintain scala version upgrade.  Even the kafka consumers are re-written in Java now.